pkg/lib/authenticationflow/accept.go

```diff
@@ -131,26 +138,24 @@ func accept(ctx context.Context, deps *Dependencies, flows Flows, inputFn func(i
 
 			switch errBotProtectionVerification.Status {
 			case ErrorBotProtectionVerificationStatusSuccess:
-				result = &AcceptResult{
-					BotProtectionVerificationResult: &BotProtectionVerificationResult{
-						Outcome: BotProtectionVerificationOutcomeVerified,
-					}}
+				result.BotProtectionVerificationResult = &BotProtectionVerificationResult{
+					Outcome: BotProtectionVerificationOutcomeVerified,
+				}
 			case ErrorBotProtectionVerificationStatusFailed:
 				// We still consider the flow has something changes.
 				changed = true
-
-				return &AcceptResult{
-					BotProtectionVerificationResult: &BotProtectionVerificationResult{
-						Outcome: BotProtectionVerificationOutcomeFailed,
-					}}, botprotection.ErrVerificationFailed
+				result.BotProtectionVerificationResult = &BotProtectionVerificationResult{
+					Outcome: BotProtectionVerificationOutcomeFailed,
+				}
+				return result, botprotection.ErrVerificationFailed
```

Comment:
This function uses named return value, so please use named return value consistently. Mixing them is a source of confusion.

Reply:
Maybe I will change it to unnamed return values. I found it difficult to understand what is the return value when there is a loop

Reply:
I found that this function must be using named return value because the defer function will change the return value, so I will change it back to named return values.